src/config/index.ts

```diff
@@ -0,0 +1,40 @@
+export const TIME_UNITS_IN_SECONDS: Record<string, number> = {
```

Comment:
separating this is nice idea, but since it's part of specific model, maybe file should be in the same folder with model. What you think?